oxenc/bt_producer.h

```diff
@@ -86,6 +88,14 @@ namespace detail {
                 std::is_same_v<decltype(result), char*> ||
                 std::is_same_v<decltype(result), const char*>)
             return std::string_view{result};
+        else if constexpr (
+                std::is_same_v<decltype(result), unsigned char*> ||
+                std::is_same_v<decltype(result), const unsigned char*>)
+            return std::basic_string_view<unsigned char>{result};
+        else if constexpr (
+                std::is_same_v<decltype(result), std::byte*> ||
+                std::is_same_v<decltype(result), const std::byte*>)
+            return std::basic_string_view<std::byte>{result};
```

Comment:
These seem highly unlikely to ever be needed/wanted: the check for `char*` is to allow null-terminated C string literals to be returned (which is mainly for test code), but something returning a null-terminated `unsigned char*` or `std::byte*` seems much more likely to be a bug than intended.